homeassistant/components/awair/const.py

```diff
@@ -86,7 +86,7 @@ class AwairSensorEntityDescription(SensorEntityDescription, AwairRequiredKeysMix
     ),
     AwairSensorEntityDescription(
         key=API_VOC,
-        icon="mdi:cloud",
+        device_class=SensorDeviceClass.VOLATILE_ORGANIC_COMPOUNDS,
         native_unit_of_measurement=CONCENTRATION_PARTS_PER_BILLION,
```

Comment:
This device class currently mandates ug/m3 as unit:
https://developers.home-assistant.io/docs/core/entity/sensor#available-device-classes